src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -48,6 +49,8 @@ import javax.swing.tree.TreePath
 @Service
 class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
 
+    private val myAlarm = Alarm()
```

Comment:
Just as a suggestion: maybe rename it to something like scrollPaneAlarm to add more context to name.

Reply:
done
PS In Idea source base it quite normal for inner objects to be named like my<Classname> and I found it helpful to get the idea about those objects scope while debugging.